src/main/webapp/trip/tripDetails.html

```diff
@@ -2,46 +2,126 @@
+                        <input type="radio" class="radio" id="jack" name="participant" />
+                        <label for="jack">Jack</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="jess" name="participant" />
+                        <label for="jess">Jess</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="mauricio" name="participant" />
+                        <label for="mauricio">Mauricio</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="nikki" name="participant" />
+                        <label for="nikki">Nikki</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="shaheen" name="participant" />
+                        <label for="shaheen">Shaheen</label>
+                      </div>
+            
+                      <div class="option">
+                        <input type="radio" class="radio" id="music" name="category" />
+                        <label for="music">Yifan</label>
+                      </div>
+                    </div>
+        
+                    <div class="selected" onclick="select()">
+                      Select a Participant
+                    </div>
+                  </div> -->
+
+                <h3 class="trip-item-or">or</h3>
+                <form action="/NewPeople" method="POST">
+                    <div class="inputBox">
+                        <input name="text-input-participants" type="text" placeholder="e.g. Patrick">
+                        <input type="submit" value="add">
+                        
+                    </div>
+                </form>
+
+                <!-- Point of interest -->
+                <h3 class="trip-item">💚 Add points of interest</h3>
+                <div class="select-box">
+                    <div class="options-container">
+                      <div class="option">
+                        <input type="radio" class="radio" id="bc" name="visit"/>
+                        <label for="bc">Yoho Natioanl Park</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="alberta" name="visit" />
+                        <label for="alberta">Banff National Park</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="qc" name="visit" />
+                        <label for="qc">Quebec City</label>
+                      </div>
+                    </div>
                     
+                    <div class="selected" onclick="select()">
+                      Select a place to visit
```

Comment:
I think it's better if user's just name their Trips to represent where they're going. It could be limiting to have to choose from a selection of places.

Reply:
This is is just to add the "event" name

Reply:
I think this should just be a text box where you can enter the string.